src/pymedphys/dcm/_level2/dcmanonymise.py

```diff
@@ -64,21 +64,12 @@ def anonymise_dicom(dcm, delete_private_tags=True, tags_to_keep=None,
     Returns
     -------
     dcm_out
-        An anonymised copy of the input DICOM file as a `pydicom FileDataset`
-
-    Raises
-    ------
-    TypeError
-        If `dcm` is not an instance of `pydicom.dataset.FileDataset`
+        An anonymised copy of the input DICOM file as a `pydicom Dataset`
     """
 
     if tags_to_keep is None:
         tags_to_keep = []
 
-    if not isinstance(dcm, pydicom.dataset.FileDataset):
-        raise TypeError("The input argument is a member of {}. "
-                        "It must be a pydicom FileDataset.".format(type(dcm)))
```

Comment:
This excludes the ability of using `pydicom.Dataset`. Is it okay if I remove this raise?

Reply:
I also believe that `pydicom.FileDataset` is a superset of `pydicom.Dataset`...

Reply:
Yep no worries at all. I added that before I had a clear understanding of pydicom's structure.